pkg/realizer/deliverable/errors.go

```diff
@@ -20,8 +20,11 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	"github.com/vmware-tanzu/cartographer/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/cartographer/pkg/utils"
 )
 
+const NO_JSONPATH_CONTEXT = "<no jsonpath context>"
```

Comment:
oh, we usually go with PascalCase rather than UPPER_CASE_WITH_UNDERSCORES for constants

```
pkg/apis/v1alpha1/cluster_supply_chain.go
29:const (
30-     SupplyChainReady          = "Ready"
31-     SupplyChainTemplatesReady = "TemplatesReady"
--
34:const (
35-     ReadyTemplatesReadyReason    = "Ready"
36-     NotFoundTemplatesReadyReason = "TemplatesNotFound"
```